xsd/netex_framework/netex_responsibility/netex_versionFrame_version.xsd

```diff
@@ -117,11 +117,6 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Default values to use on elements in the frame  that do not explicitly state a value.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="versions" type="versions_RelStructure" minOccurs="0">
```

Comment:
didn't we agree to deprecate ? 
I  don't have my notes with me, but this may break quite a lot a existing data

Reply:
In _next_ we don't have to deprecate right? That is what should happen in master.

Reply:
that's not my understanding... we decided to use deprecate multiple times to limits the incompatibilities. And for version, the decision of the group that I wrote on my notes is "deprecate"

Reply:
no, my understanding is that the group wanted to deprecate on 'next' (we already did multiple deprecation on next) in order to give time to update production. We will then the remove in 3.0

Reply:
https://github.com/NeTEx-CEN/NeTEx/issues/641

Reply:
It is fine by me. We have assured in this pull request that all examples are stripped of the Version element, we can 'revert' the removed elements.